chart/githubconnector/templates/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
```

Comment:
Don't put namespace in resources managed by helm. The user provides the namespace when he installs the chart, so no need to put it here. In fact, you will break the release if the namespace will be different then the one provided by the user.